mlrun/api/initial_data.py

```diff
@@ -537,48 +537,72 @@ def _enrich_project_state(
             db.store_project(db_session, project.metadata.name, project)
 
 
+def _delete_default_hub_source(db_session: sqlalchemy.orm.Session):
```

Comment:
IMO it's easier to read code top to bottom, meaning that the called method is below the caller.
It's easier to understand the flow without having to guess what's the end goal of the helper methods.
It seems that this file is written the opposite so if it's on purpose we can discuss but if not, please flip the order.